frontend/src/services/Rpc/solana/RpcServiceSolana.ts

```diff
@@ -64,12 +244,24 @@ export class RpcServiceSolana implements IRpcService, ITokenService {
     return this.sendTransaction(hex);
   }
 
-  private async fetchRecentBlockhash(): Promise<string> {
+  async fetchRecentBlockhash(): Promise<string> {
     const requestBody = {
       jsonrpc: '2.0',
       id: 1,
       method: 'getLatestBlockhash',
-      params: [{ commitment: 'finalized' }],
+      params: [{ commitment: 'finalized' }], //Official Solana recommendation: Use confirmed commitment when fetching recent blockhash. This will provide a better (extended with aprox. 13s) time window as compared to finalized commitment, thus reducing the risk of transaction expiration (see below).
+    };
+
+    const response = await this.postRequest(rpcURL, requestBody);
+    return response.result?.value?.blockhash as string;
+  }
+
+  static async fetchRecentBlockhash(): Promise<string> {
```

Comment:
why we have to function with exactly the same name here?

Reply:
I was testing it. To try to call it on while signing. So I had to access it externally.